index.html

```diff
@@ -232,16 +234,29 @@
     email = email.toLowerCase();
     popupContainer.innerHTML = '<div class="teacher"><div>' + '<img src="' + imgSrc +
         '" align="right" class="fullsizeimage"></img><p>' + teacherObj.bio + '</p></div>' +
-        '<schedule-lite></schedule-lite></div>';
-    renderSchedule(globalDate, teacherObj, "teacher")
+        '<schedule-lite id="teacherschedule"></schedule-lite></div>';
+    var scheduleElement = document.getElementById('teacherschedule');
+    renderSchedule(globalDate, teacherObj, "teacher", scheduleElement)
   }
   function renderStudent(studentObj) {
     console.log(studentObj);
+    var name = studentObj.firstname + " " + studentObj.lastname;
+    var email = studentObj.firstname[0] + studentObj.lastname + "@eastsideprep.org";
+    email = email.toLowerCase();
+    var grade = studentObj.grade + "th Grade";
+    var popupContainer = document.getElementById("popupContainer");
+    popupContainer.innerHTML = '<div class="teacher"><div>' + '<img src="/images/material_face.svg"' +
+        ' width="32" height="32"><h2>' + name + '</h2><h3><a href="mailto:' + email + '">' +
```

Comment:
Fixed in <a href="https://github.com/guberti/epschedule/commit/42384c202411de1c6e969a62b06cf69f9caf39ba">Don't show name twice on student view</a>